docs/DeveloperGuide.adoc

```diff
@@ -797,73 +873,414 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a/an ... |I want to ... |So that I can...
+|`* * *` |New member |see usage instructions |refer to instructions when I forget how to use the App
+
+|`* * *` |<<exco-member,Exco member>> |add a new member to the club | begin assigning groups and tasks to the member
+
+|`* * *` |Exco member |delete a member from the club |remove entries that the club no longer needs
+
+|`* * *` |Exco member |divide members into groups using tags |manage the committee members more efficiently
+
+|`* * *` |Exco member |remove a particular tag/group | remove redundancies or remove irrelevant tags
+
+|`* * *` |Exco member |remove a member from a particular group | keep the App up-to-date, and also take into account students that opt-out or graduate
 
-|`* * *` |user |add a new person |
+|`* * *` |Exco member |add a task for certain members | assign members to do a task
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |Exco member |view the status of a task | see the progress made towards completing the task
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |Exco member |remove a task | delete tasks that have been completed
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |Exco member |create a poll | get the opinions of the club members
+
+|`* * *` |Exco member who created a poll |view results of the poll | see the opinions of the members
+
+|`* * *` |Member |edit my contact details |keep my contact information updated
```

Comment:
Is there a difference between `Exco member` and `Member`? Standardize the naming conventions if it's not